Cargo.toml

```diff
@@ -19,9 +18,11 @@ lettre_email = "0.9"
 mime_guess = "2"
 ratatui = { version = "0.20", optional = true }
 chrono = "0.4"
+clap = { version = "3", features = ["derive"], optional = true }
 
 [features]
-default = ["tui"]
+default = ["tui", "cli"]
+"cli" = ["dep:clap"]
 "tui" = ["dep:crossterm", "dep:ratatui"]
```

Comment:
Can we try removing these annoying `"`?
I was lazy and let copilot fill these for me...

```suggestion
cli = ["dep:clap"]
tui = ["dep:crossterm", "dep:ratatui"]
```